CHANGELOG.md

```diff
@@ -1,9 +1,34 @@
+## 3.0.14
```

Comment:
let's rise minor version since there are feature updates, and not bug fix patching only.

Reply:
Raised the version to 3.1.0.0.